src/main/telemetry/srxl.c

```diff
@@ -160,6 +162,174 @@ bool srxlFrameRpm(sbuf_t *dst, timeUs_t currentTimeUs)
     return true;
 }
 
+#if defined(USE_GPS)
+
+// From Frsky implementation
+static void GPStoDDDMM_MMMM(int32_t mwiigps, gpsCoordinateDDDMMmmmm_t *result)
+{
+    int32_t absgps, deg, min;
+    absgps = ABS(mwiigps);
```

Comment:
Please do not align the `=`. If done, this will result in unneeded change whenever a new assignment to a variable with a longer name is added.

Reply:
This is actually not my code, but a function taken directly from the existing frsky_hub.c
Of course, I don't see any problem with changing it.